linux-tkg/README.md

```diff
@@ -0,0 +1,70 @@
+**Due to intel_pstate poor performances as of late, I have decided to set it to passive mode to make use of the acpi_cpufreq governors passthrough, keeping full support for turbo frequencies.**
+
+### PDS, MuQSS and BMQ are not yet available options for 5.9 RC
+### MuQSS is not an available option for 5.8.y yet
+## Nvidia prop drivers need to be patched (https://github.com/Frogging-Family/nvidia-all can do that automatically for you)
+
+
+A custom Linux kernel (5.9 RC, 5.8.y, 5.7.y, 5.4.y) with added tweaks for a nice interactivity/performance balance, aiming for the best gaming experience.
+
+MuQSS : http://ck-hack.blogspot.com/
+
+Project C / BMQ : http://cchalpha.blogspot.com/
```

Comment:
Project C / PDS should also be added here now. And I have also changed the note below regarding "undead PDS". See https://github.com/Frogging-Family/linux-tkg/commit/15169ce8485257f37d15bd639b8144e183cb3473#diff-0230580de75edb6209f4524aac56c0c3

Reply:
Is the new change okay?